### ISSUE TITLE
New GitHub org: kubeflow-wgs

### ISSUE THREAD

**Issue by thesuperzapper** [kubeflow/community#456]
Given there is discussion about moving away from the centralised [kubeflow](https://github.com/kubeflow) GitHub org, I have registered [kubeflow-wgs](https://github.com/kubeflow-wgs).

I propose that this new org will be controlled by a panel of people from the established Working Groups. I believe this approach is preferable to fragmenting into our own GitHub organisations.

For reference, kubernetes has something similar with their [kubernetes-sigs](https://github.com/kubernetes-sigs) GitHub org.

Thoughts?

**Comment by jlewi** [comment]
@thesuperzapper  thanks for opening this unfortunately  I think this is problematic. Please go back to [Chris' email](
https://groups.google.com/g/kubeflow-discuss/c/v6iS6F_iZSI/m/RTWodi8DBgAJ)

I would not recommend people use this new GitHub org as that seems likely to cause problems down the line. 

If you want to setup new governance structures I think per Chris' email then you need to create a proper fork. As Chris says I don't think this is a bad thing. If projects need to fork to be successful I encourage them to do so and the two communities can continue to collaborate.

@thesuperzapper I think it would be best if you deleted the new GitHub org.  Per kubeflow/kubeflow#5418 I'm very supportive of you creating a new GitHub org but I recommend avoiding names like "Kubeflow-<foo>" aside from the issues named above I think its just boring. I think there's a lot more descriptive and interesting names you could come up with.

/close
/cc @kubeflow/project-steering-group

**Comment by thesuperzapper** [comment]
@jlewi  this proposal is not about creating a new governance structure for Kubeflow, but addressing the fact Google is unwilling to maintain the `kubeflow` GitHub org from an engineering perspective.

Nothing about moving the repositories (which are owned by the working group leads) into `kubeflow-wgs` will affect the current governance structure, it just gives the code a home.

All the governance stuff will still live on the `kubeflow` GitHub org.

Centralising the code (not governance) on `kubeflow-wgs` will also reduce the overhead for [AWS's CI proposal](http://bit.ly/kubeflow-test-infra-aws), as otherwise they would have to support many other github orgs (one for each working group).

@Jeffwan thoughts from an AWS perspective? 

@kubeflow/google-admins thoughts from an engineering perspective?

@kubeflow/project-steering-group (I am pretty sure this group doesn't exist, both in GitHub, and reality)

**Comment by PatrickXYS** [comment]
Let me clarify something first, from our AWS side; we'd like to provide an **optional-test-infra** for kubeflow WG.

We'll focus on the engineering work afterwards, and leave governance and steering work for Google.

**Comment by Bobgy** [comment]
> @kubeflow/project-steering-group (I am pretty sure this group doesn't exist, both in GitHub, and reality)

@thesuperzapper I've fixed the configuration and it's now visible from public. I speak for this group, but there's nothing decided that I can share publicly right now. The latest update is what I shared in last community meeting, that we are working on trademark guidance, so before that's settled, I'd recommend not creating any new org in the name of `kubeflow`.
